### DISCUSSION TITLE
FluidUI web dropdown doesn't scroll using touch

### DISCUSSION THREAD

**Discussion by alonsh4** [microsoft/fluentui#16328] [Q&A] [answered]
Seems the FluentUI React Web 'Dropdown' component doesn't support scroll using touch
Repro:
1. Open web application with a FluentUI dropdown control, on a touch device (like windows laptop)
2. Open dropdown in web. If it's not long enough to require scroll - reduce the screen resolution
3. Using touch screen - try to scroll through the list - nothing happens
4. Try to scroll by touching the scrollbar - nothing happens either

Is this a known issue? Any plans to change it?

Alon Shmueli
alonsh@microsoft.com

**Comment by JustSlone** [comment, marked as answer]
@alonsh4 you might be hitting this issue: #15750. If not, please file an issue on this. 

We'd like to keep Discussions for questions and conversation rather than bug reports.
